api/v1alpha1/cdapmaster_types.go

```diff
@@ -210,6 +218,11 @@ type SupportBundleSpec struct {
 	CDAPStatefulServiceSpec `json:",inline"`
 }
 
+//  SystemMetricExporterServiceSpec defines the specification for the SystemMetricsExporter service.
+type SystemMetricExporterServiceSpec struct {
+	CDAPScalableServiceSpec `json:",inline"`
```

Comment:
Do we need the complete service spec here?

Reply:
We need properties for resource requests and limits. Other properties such as ServiceAccount, Env, Configmaps may be required in the future. Changed to inherit from CDAPServiceSpec as replica count wasn't useful here.